app/build.gradle

```diff
@@ -79,6 +80,7 @@ android {
             buildConfigField "String", "CLIENT_ID", "\"${System.getenv("CLIENT_ID")}\""
             buildConfigField "String", "CLIENT_SECRET", "\"${System.getenv("CLIENT_SECRET")}\""
             buildConfigField "String", "REDIRECT_URI", "\"${System.getenv("REDIRECT_URI")}\""
+            buildConfigField "String", "FEEDBACK_DB_PATH", "\"feedback\""
```

Comment:
change it to "\"${System.getenv("FEEDBACK_PATH")}\""
I'll add it through the secrets in github actions